x/txfees/keeper/feedecorator.go

```diff
@@ -62,7 +62,8 @@ func (mfd MempoolFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate b
 	}
 
 	// TODO: In v26, move this code to the correct spot, and make it CheckTX only.
-	if ctx.BlockHeight() >= 17004043 {
+	bh := ctx.BlockHeight()
+	if bh <= 16841115 || bh >= 17004043 {
```

Comment:
Could we separate this into a separate PR so that this change is easier to track for historical purposes with its own PR description?

Reply:
Make sense. Created separate PR for forward porting it to v26, and let mergify to kick in and merge the changes into v25.

PR: https://github.com/osmosis-labs/osmosis/pull/8629